python/lsst/obs/subaru/crosstalk.py

```diff
@@ -511,16 +513,16 @@ def fixCcd(butler, visit, ccd, coeffs, display=True):
     """Apply cross-talk correction to a CCD, given the cross-talk coefficients"""
     mi = readImage(butler, visit=visit, ccd=ccd)
     if display:
-        ds9.mtv(mi.getImage(), frame=0, title="CCD %d" % ccd)
+        frame1 = afwDisplay.getDisplay(backend="ds9", frame=1)
```

Comment:
It looks as though we previously put the pre-subtraction image in frame `0` and the post-substraction in frame `1`. You seem to be putting both in `1`. Is that deliberate?